Vagrantfile

```diff
@@ -64,6 +64,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Can be removed once PR #11528 with the golangci-lint replacement is merged.
```

Comment:
Nit: Since we don't plan to backport #11528 to v1.8, I think we should drop this comment.

Reply:
Let's keep it so we don't lose context.